service.subtitles.pipocas/service.py

```diff
@@ -347,31 +398,39 @@ def Search(item):
         subtitles_list = getallsubs(searchstring, "en", "English", file_original_path, searchstring_notclean)
         for sub in subtitles_list: append_subtitle(sub)
     if PT_ON == 'false' and PTBR_ON == 'false' and ES_ON == 'false' and EN_ON == 'false':
-        xbmc.executebuiltin((u'Notification(%s,%s,%d)' % (_scriptname , 'Apenas Português | Português Brasil | English | Spanish.',5000)))
+        xbmc.executebuiltin((u'Notification(%s,%s,%d)' % (_scriptname , normalizeString('Apenas Português | Português Brasil | English | Spanish.'),5000)))
```

Comment:
xbmc.Dialog().notification() please. Avoid using builtins when the same functionality is available in the python api